public/app/features/plugins/partials/ds_http_settings.html

```diff
@@ -94,6 +95,24 @@ <h6>Basic Auth Details</h6>
   </div>
 </div>
 
+<div class="gf-form-group" ng-if="current.addCustomHeaders">
+  <div ng-controller="current.addHeader">
```

Comment:
this controller does not exist. So it raises an error. I don't think we should create a new controller for this. Just reuse the `DataSourceEditCtrl`